README.md

```diff
@@ -1,2 +1,43 @@
-# RustDiscordBot
-Introductory project on using rust making some kind of advanced discord bot
+
+# Project Title
+
+Hello World Initial Project
+
+## Installation
+
+Go to this website to install rust
+https://www.rust-lang.org/tools/install
+
+
+```
+```
+## Contributing
+
+Contributions are always welcome!
+
+See `contributing.md` for ways to get started.
+
+Please adhere to this project's `code of conduct`.
+
+## Running Tests
+
+tbd...
+
+```
+```
+
+## Authors
+
+- [@JarrodCarson](https://github.com/JarrodCarson)
+
+- [[@mamaluigie](https://github.com/mamaluigie)
+
+
+## Ideas on what to do
+
+- Maybe like a bot that records everything in every channel that happens in a discord server and log it into a db. we can then use this information to make a graph db and display like connections on how many times someone has talked to someone else or something? I was thinking about putting this bot into a server passively to collect data and create a graph on lol.
```

Comment:
feel free to add ideas in a cr lol

Reply:
Maybe we can do some language analysis, like analyzing the statistics of word usage, or performing some kind of language processing. This could be the interface for an AI/ML project.

Reply:
oh hell yes, also I entered an issue to the issue board about some ideas and data that we can track if we choose to do this idea